packages/documentation/src/stories/raw-components/list/list.snapshot.stories.ts

```diff
@@ -50,40 +50,3 @@ export const PostList: Story = {
     },
   ],
 };
-
-export const List: Story = {
-  render: () => {
-    return schemes(
-      () => html`
-        <ol>
-          <li>This is an ordered list.</li>
-          <li>It appears in its default style.</li>
-          <li>
-            Therefore it should be rendered with sequential numbers at the beginning of each list
-            item.
-          </li>
-          <li>
-            Nested list:
-            <ol>
-              <li>This is a nested list</li>
-              <li>It is further indented, depending on the depth of nesting.</li>
-              <li>Nested lists numbers are independent form the numbers of their parents.</li>
-            </ol>
-            After nested list item
```

Comment:
It would be good to have a text after the nested lists in the snapshots as we found a bug with this specific use case when developing the ordered lists.